libcaliptra/test/if_test/if_test_impl.c

```diff
@@ -0,0 +1,31 @@
+//Licensed under the Apache-2.0 license
```

Comment:
We probably need an implementation of this that uses the hw-model; are you planning on adding this in a future PR?

Reply:
Yes, I'm working with Fabrizio to move this in that direction.